lightwood/api/json_ai.py

```diff
@@ -270,6 +266,7 @@ def generate_json_ai(
     elif list(outputs.values())[0].data_dtype in [dtype.array]:
```

Comment:
seems like `list(outputs.values())[0].data_dtype` is called earlier. might as well make this a var as otherwise since it's gonna get called. I don't know if this is expensive or a moot point to do tho but it does reduce the number of times we need to evaluate this.